source/features/add-selection-in-new-tab.js

```diff
@@ -11,6 +11,11 @@ export default function () {
 				urls: [selected.href],
 				action: 'openAllInTabs'
 			});
+
+			// Get the list element that contains the unread class and mark it as read.
+			const classList = selected.parentElement.parentElement.classList;
```

Comment:
Open to a different way to jump from the `<a>` back to the `<li>`.

Reply:
`.closest()`